.github/workflows/build-and-test.yml

```diff
@@ -14,7 +14,7 @@ concurrency:
 
 jobs:
   build-and-test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
My build script has been hanging for a long time, so I have split the build file now.

Reply:
Could you rebase or merge with HEAD? It's already updated there.

Reply:
> Could you rebase or merge with HEAD? It's already updated there.

Yes, later I will rebase and squash the PR I'm not trying to merge the file; it is to test the build system. Somehow, the build system is not working for this PR.